[2025-06-03T20:29:46.697+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table manual__2025-06-03T20:29:43.535627+00:00 [queued]>
[2025-06-03T20:29:46.713+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table manual__2025-06-03T20:29:43.535627+00:00 [queued]>
[2025-06-03T20:29:46.714+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2025-06-03T20:29:46.735+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-06-03 20:29:43.535627+00:00
[2025-06-03T20:29:46.744+0000] {standard_task_runner.py:60} INFO - Started process 62903 to run task
[2025-06-03T20:29:46.750+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'delete_data_from_table', 'manual__2025-06-03T20:29:43.535627+00:00', '--job-id', '370', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_dag.py', '--cfg-path', '/tmp/tmp7m9fxbnc']
[2025-06-03T20:29:46.753+0000] {standard_task_runner.py:88} INFO - Job 370: Subtask delete_data_from_table
[2025-06-03T20:29:46.876+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.delete_data_from_table manual__2025-06-03T20:29:43.535627+00:00 [running]> on host 4bea2aab752f
[2025-06-03T20:29:46.988+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-03T20:29:43.535627+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-03T20:29:43.535627+00:00'
[2025-06-03T20:29:46.990+0000] {sql.py:276} INFO - Executing: 
            delete from dag_runs where dt = '2025-06-03' and dag_id = 'dag_with_postgres_operator_v01';
        
[2025-06-03T20:29:47.006+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-03T20:29:47.023+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-03T20:29:47.066+0000] {sql.py:450} INFO - Running statement: 
            delete from dag_runs where dt = '2025-06-03' and dag_id = 'dag_with_postgres_operator_v01';
        , parameters: None
[2025-06-03T20:29:47.081+0000] {sql.py:459} INFO - Rows affected: 1
[2025-06-03T20:29:47.100+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=delete_data_from_table, execution_date=20250603T202943, start_date=20250603T202946, end_date=20250603T202947
[2025-06-03T20:29:47.165+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-03T20:29:47.196+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
